.evergreen/config.in.yml

```diff
@@ -107,9 +107,9 @@ functions:
         script: |
           ${PREPARE_SHELL}
 
-          # TODO(NODE-5035): Remove when merged - need to replace with branch just for OIDC.
+          # TODO(NODE-5035): Remove these 2 lines when PR merged.
           rm -rf "${DRIVERS_TOOLS}"
-          git clone --branch DRIVERS-2415 https://github.com/blink1073/drivers-evergreen-tools.git "${DRIVERS_TOOLS}"
+          git clone --branch DRIVERS-2415-orch https://github.com/mongodb-labs/drivers-evergreen-tools.git "${DRIVERS_TOOLS}"
```

Comment:
Just leaving a comment to remind about this

Reply:
Removed as PR is merged.